fusor-ember-cli/app/templates/configure-environment.hbs

```diff
@@ -16,8 +16,8 @@
       Select how this deployment's instances should receive updates.
     </p>
     <div class='ident-radio'>
-      {{#radio-button value="immediately" groupValue=envSetup changed="envSetupChanged" id='immediately' disabled=deploymentController.isStarted}}
-        <span class="{{if deploymentController.isStarted 'disabled'}}">
+      {{#radio-button value="immediately" groupValue=envSetup changed="envSetupChanged" id='immediately' disabled=isStarted}}
```

Comment:
the *new* deployment was picking up the `isStarted` value on the `deploymentController` from the previous deployment which was `isStarted`.